shared-bindings/gnss/SatelliteSystem.c

```diff
@@ -10,30 +10,20 @@
 //|     def __init__(self) -> None:
 //|         """Enum-like class to define the satellite system type."""
 //|
-//|     GPS: SatelliteSystem = ...
-//|     """Global Positioning System.
+//|     GPS: SatelliteSystem
```

Comment:
Why is the `= ...` not needed?

Reply:
It's valid with `= ...`, but:

- typeshed doesn’t add `= …` to constants and properties.
- sphinx-autoapi renders `= ...` as '*= Ellipsis*' that can be confusing.
  <img width="200" alt="= Ellipsis" src="https://user-images.githubusercontent.com/5351911/88449888-a0162700-ce85-11ea-9d49-83e6a4ba8dbc.png">

So I think just removing them is reasonable.

Reply:
Totally agree! Is there anyway we could check that the `= ...` isn't added?